### ISSUE TITLE
Put label-same-as-description (from label / select-name) into its own rule

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#1325]
The help-same-as-label check is the only check in axe-core disabled by default. This is confusing. We should either put it in its own rule, have it run as a best practice, or we should remove it.

**Comment by dylanb** [comment]
Should be a best practice rule

**Comment by WilcoFiers** [comment]
The check for this is called `help-same-as-label`. This is kind of a confusing check ID, so I'm going to use a different rule ID for this.

**Comment by straker** [comment]
You'll need to create a custom rule to use it since we don't have any rules that use it by default anymore.

```html
<input title="Username" aria-label="Select Username">
<script>
axe.configure({
  rules: [
    {
      id: 'help-same-as-label',
      any: ['help-same-as-label']
    }
  ]
});

await axe.run()
</script>
```

**Comment by padmavemulapati** [comment]
@straker , thank you , it is working with custom-rule configuration for `help-same-as-label` rule. 



https://github.com/dequelabs/axe-core/assets/47409579/a4a92708-07a1-4033-b0dc-c0a4edc3ad84

**Comment by somaalapati** [comment]
@padmavemulapati Please add environment details in the closing comments.

**Comment by padmavemulapati** [comment]
> @padmavemulapati Please add environment details in the closing comments.

@somaalapati , axe-core tickets validating on the latest develop branch code base, and adding the `axe.js` file in  source - `snippets` running the source to `axe.run()` so obviously it is testing in chrome browser - developer tool.
